scripts/kubecf-apply.sh

```diff
@@ -53,9 +56,19 @@ fi
 if [ -z "${CHART:-}" ]; then
     CHART="output/kubecf-$(./scripts/version.sh).tgz"
     export TARGET_FILE="${CHART}"
+    if [ -n "${RENDER_LOCAL:-}" ]; then
```

Comment:
What does `RENDER_LOCAL` mean? Can you render remotely?
Adding a new concept of `NO_IMAGELIST` - could just check for `RENDER_LOCAL`.
This adds 2 new concepts, I think they are both hard to understand intuitively. `RENDER_LOCAL` should be renamed and `NO_IMAGELIST` removed.

Reply:
I used `RENDER_LOCAL` because Andreas had created a copy of `kubecf-build.sh` called `kubecf-render-local, with the only different being that it called `helm template` instead of `helm upgrade`. I changed it to have an env setting to control this instead, to avoid the code duplication.

It is a feature for development/debugging of the templates.

Same thing for `NO_IMAGELIST`. During development I often insert a variable dump at a strategic place:

```
{{- include "_config.fail" (toYaml $.Values) }}
```

(or just some subtree that I'm interested in). This obviously breaks image_list.rb, which runs the templates internally, so I don't get a working chart that I can test with my own `--set` and `--values` overrides in `kubecf-render-local`. For this reason I object to removing the `NO_IMAGELIST` config variable.

Re-reading this I realize you want to just re-use one variable for both purposes. Will think about this on Wednesday when I'm back at work...